support-frontend/nginx/support.conf

```diff
@@ -1,8 +1,7 @@
 server {
```

Comment:
nginx was updated and this is the new syntax, using the old one throws deprecation warnings on open